### PR TITLE
build: fix onebranch pipelines

### PR DESCRIPTION
**Pull request by jianye-chen**
## Description

Follow-up PR to address the onebranch pipeline failures introduced by #3343.  
  
`gamecore_console` builds need to be on SDK `10.0.20348.0` or later and the latest SDK in our existing image is `10.0.19041.0`.  
The old base image (`cdpxwin1809.azurecr.io/global/vse2019:latest`) had some issues with installing the new SDK, so we've changed to a more recent onebranch base image for our builds.  
Also adding a new pipeline parameter to specify the base image tag to use, so we could test the official build against wip images. Defaulting to `latest` instead of `20220812.8`.  
  
Thanks to @csujedihy for helping out!  

## Testing

Through pipeline runs.  
`msquic-Official-Docker #20230113.3` publishing to `testing`, and subsequent `msquic-Official #20230113.2` based on the `testing` image.  
Waiting for the publish to `latest` in `msquic-Official-Docker #20230113.4`.  

## Documentation

N/A

### CODE REVIEW

File: .azure/OneBranch.Official.yml
Comment (csujedihy) [inline .azure/OneBranch.Official.yml:30]: Have you already pushed the image with the tag "latest" to the isolated ACR?

Comment (jianye-chen) [reply .azure/OneBranch.Official.yml:30]: I have pushed the `testing` image. The `latest` image is currently being built and pushed in `msquic-Official-Docker #20230113.4`.

Comment (csujedihy) [reply .azure/OneBranch.Official.yml:30]: Cool, thanks, signing off. I will leave it up to decice whether check it in now or wait for `latest` to finish uploading.

**Review by csujedihy — APPROVED** [review]

**Comment by csujedihy** [comment]
Waiting for the onebranch docker pipeline to finish uploading. Will merge after that.

**Comment by csujedihy** [comment]
@nibanks this one can go in now. Need your help to override the PR gate.

**Pull request merged**
